src/main/java/io/parsingdata/metal/data/selection/ByToken.java

```diff
@@ -37,4 +38,19 @@ public static ParseItem get(final ParseGraph graph, final Token definition) {
         return get(graph.tail, definition);
     }
 
+    public static ParseItemList getAll(final ParseGraph graph, final Token definition) {
+        if (definition == null) { throw new IllegalArgumentException("Argument definition may not be null."); }
+        return getAllRecursive(graph, definition);
+    }
+
+    private static ParseItemList getAllRecursive(final ParseGraph graph, final Token definition) {
```

Comment:
@ccreeten: The implementation of this method is subtly different from the similar one in ByName. Can you explain why you've chosen this approach?